ui/app/models/pki/config/acme.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
+
+@withFormFields()
+export default class PkiConfigAcmeModel extends Model {
+  // This model uses the backend value as the model ID
+  get useOpenAPI() {
+    return true;
+  }
+
+  getHelpUrl(backendPath) {
+    return `/v1/${backendPath}/config/acme?help=1`;
+  }
+
+  // attrs order in the form is determined by order here
+
+  @attr('boolean', {
+    label: 'ACME enabled',
+    subText: 'When ACME is disabled, all requests to ACME directory URLs will return 404.',
+  })
+  enabled;
+
+  @attr('string', {
+    subText:
+      'Optionally specifies a role to enforce on the default ACME directory. Must exist in allowed_roles if set.',
+  })
+  defaultRole;
```

Comment:
\o @hellobontempo https://github.com/hashicorp/vault/pull/20935 just merged; would you mind updating this to `default_directory_policy`, please? Thank you!!

Reply:
done! and updated `allowed_roles` subtext